HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -2775,10 +2775,34 @@ def self.create_curve_quint_linear(model, coeff, name)
     curve.setCoefficient6z(coeff[5])
     return curve
   end
+  
+  def self.convert_rated_to_gross_capacity_cop_clg(clg_ap)
+    clg_ap.detailed_performance_net_cap_clg.keys().sort.each do |out_db|
+      clg_ap.detailed_performance_net_cap_clg[out_db].each_with_index do |cap, i|
+        net_power = cap / clg_ap.detailed_performance_net_cop_clg[out_db][i]
+        #gross_cap = cap + clg_ap.fan_powers_rated
```

Comment:
@shorowit @jonwinkler resdx is using pretty complicated approach to calculate the cooling_rated_fan_power for each speed,   it uses free efficacy, efficacy slope and the operating pressure to calculate power. See [here](https://github.com/bigladder/resdx/blob/main/resdx/neep_vchp.py#L126), [here](https://github.com/bigladder/resdx/blob/main/resdx/fan.py#L409-L412), and [here](https://github.com/bigladder/resdx/blob/main/resdx/fan.py#L91-L98). 

There're a bunch of function calls, and I don't think we have anything that complicated in our workflow yet. In writing FanSystemModel E+ object, we are using power_fraction = fan_ratio**3 as fan power curve. Does it sound reasonable to use `power_fraction = fan_ratio**3` to calculate the fan power at each stages?
For cfm/ton, we still keep the assumption from previous work, right? With just using the first and last items in the array as min and max?

Reply:
Proceed with the power_fraction = fan_ratio**3 curve